packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.template.ts

```diff
@@ -0,0 +1,46 @@
+import { html, ref } from "@microsoft/fast-element";
+import { endTemplate, startTemplate } from "../../patterns/start-end";
+import { AccordionItem } from "./accordion-item";
+
+export const AccordionItemTemplate = html<AccordionItem>`
+    <template
+        class="${x => (x.expanded ? "expanded" : "")}"
+        aria-expanded="${x => x.expanded}"
```

Comment:
@eljefe223 Is this aria-expanded redundant? I don't see this called out here: https://w3c.github.io/aria-practices/#accordion